### PR TITLE
docs: Links between docs

### PR DESCRIPTION
**Pull request by MarekPieta**
Change adds missing links between docs for the Event Manager and Profiler.

### CODE REVIEW

**Comment by MarekPieta** [comment]
@ru-fu, could you review ?

File: samples/profiler/README.rst [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
 It shows how to use the Profiler to log and visualize data about custom events that are not part of the :ref:`event_manager`.
 
 Overview
 ********
 
-The sample initializes the Profiler, registers two event types, and profiles their occurrences periodically.
+The sample initializes the :ref:`profiler`, registers two event types, and profiles their occurrences periodically.
 
 Event without data (``no data event``):
   This event is used to signal the occurrence of an event only.
```
Comment (ru-fu) [inline samples/profiler/README.rst]: There's another occurrence further down in the Dependencies section.

Comment (MarekPieta) [reply samples/profiler/README.rst]: fixed

**Comment by MarekPieta** [comment]
@pdunaj, can we merge it ?

**Pull request merged**